Package.swift

```diff
@@ -50,13 +50,9 @@ let package = Package(
       path: "Sources"
     ),
     .testTarget(
-      name: "FirebaseDataConnectTests",
```

Comment:
Should there be another target for the Integration tests, so they can continue to be run locally?

Reply:
I was going to add that later once we know Unit tests are running in git.